builders/build/templates/substrate.md

```diff
@@ -0,0 +1,71 @@
+
+1. [Incorporating pre-existing Substrate built-in modules](/builders/build/local/adding-built-in-pallet/){target=_blank} into the runtime
+2. [Creating custom modules](/builders/build/local/adding-custom-made-module/){target=_blank} tailored to your specific application needs
+
+Since the provided template already includes the essential configurations for seamless integration into the Polkadot ecosystem and compatibility with the Tanssi protocol, teams interested in constructing an innovative Appchain can use this template as a starting point for adding their custom logic.
+
+Here are some of the features that come with this template:
+
+- Utilize Tanssi's [block production as a service](/learn/tanssi/technical-features/#block-production-as-a-service){target=_blank}
+- Use [Polkadot's finality gadget](https://wiki.polkadot.network/docs/learn-consensus#finality-gadget-grandpa){target=_blank}
+- Benefit from [Polkadot's shared security model](https://wiki.polkadot.network/docs/learn-parachains#shared-security){target=_blank}
+- Use the [Polkadot.js API](/builders/interact/substrate-api/polkadot-js-api){target=_blank} to interact with the Substrate API
+
+By leveraging these features in the template, you can kickstart your Appchain development and customize it to meet your specific requirements and innovations.
+
+## Included Modules {: #included-modules }
+
+Some of the included modules are necessary for supporting the operation of the Appchain as part of the broader Polkadot ecosystem, some other modules are included to enable the Tanssi protocol and its block production mechanism, and some other modules provide functional behavior that the users can interact with. 
+
+These are some of the functional modules exposing a behavior to the users that are included and ready to use:
+
+- **[pallet_balances](https://paritytech.github.io/substrate/master/pallet_balances/index.html){target=_blank}** - the Balances pallet provides functions for handling accounts and balances for the Appchain native currency
+- **[pallet_utility](https://paritytech.github.io/polkadot-sdk/master/pallet_utility/index.html){target=_blank}** - the Utility pallet provides functions to execute multiple calls in a single dispatch. Besides batching transactions, this module also allows the execution of a call from an alternative signed origin
+- **[pallet_proxy](https://paritytech.github.io/polkadot-sdk/master/pallet_proxy/index.html){target=_blank}** - the Proxy pallet provides functions to delegate to other accounts (proxies) the permission to dispatch calls from a proxied origin.
+- **pallet_maintenance_mode** - the Maintenance Mode pallet allows the Appchain to be set to a mode where it doesn't execute balance/asset transfers and other transactions such as XCM calls. This could be useful when upgrading the runtime in an emergency, when executing large storage migrations, or when a security vulnerability is discovered.
```

Comment:
```suggestion
- **[pallet_maintenance_mode](https://github.com/moondance-labs/moonkit/blob/tanssi-polkadot-v1.3.0/pallets/maintenance-mode/src/lib.rs){target=_blank}** - the Maintenance Mode pallet allows the Appchain to be set to a mode where it doesn't execute balance/asset transfers and other transactions such as XCM calls. This could be useful when upgrading the runtime in an emergency, when executing large storage migrations, or when a security vulnerability is discovered.
```